java/spring/container/Scripts/rollback-groups-by-tag.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+echo "Querying for production resource groups"
+groups=$(az group list --tag Environment=prod --query "[].name" --output tsv)
+for name in $groups
+do
+	echo "rolling back tags for resource group:" $name
+	az group update --name $name --tags Environment=preprod
```

Comment:
Minor thing, I don't think we want to tag this back as `preprod` because we would only rollback when an issue was found. So I think we want to tag this as `error` or something. 

If that Environment doesn't exist then maybe leave this RG as untagged? But I would ask Xinyi what to do in this case.